diff --git a/bin/mx-scripts.js b/bin/mx-scripts.js
index 569815681e83b7858681548abda14505691ab81b..39fa0f47a063e8628f64d09dbe2372e198c4b7bc 100755
--- a/bin/mx-scripts.js
+++ b/bin/mx-scripts.js
@@ -10,10 +10,10 @@ const { auditPluggableWidgetsTools } = require("../dist/commands/audit.js");
 const { prettierConfigPath } = require("../dist/utils/formatting.js");
 const { toolsRoot, widgetRoot } = require("../dist/widget/paths.js");
 
-checkNodeVersion();
+// checkNodeVersion(); // Disabled via pnpm patch
 (async () => {
     try {
-        await checkMigration();
+        // await checkMigration(); // Disabled via pnpm patch
     } catch (e) {
         console.log(red("An error occurred while checking migration dependencies: ", e));
     }
diff --git a/configs/rollup.config.mjs b/configs/rollup.config.mjs
index 346d6ee995322825d87eaa10e5aa924ca456a7cf..38924ca4d2056d62f87dc6066677d82f97a29d67 100644
--- a/configs/rollup.config.mjs
+++ b/configs/rollup.config.mjs
@@ -119,7 +119,7 @@ export default async args => {
                     extensions,
                     transpile: production,
                     babelConfig: {
-                        presets: [["@babel/preset-env", { targets: { safari: "12" } }]],
+                        presets: [["@babel/preset-env", { targets: { safari: "16.3" } }]],
                         allowAllFormats: true
                     },
                     external: outputFormat === "es" ? [] : webExternal,
@@ -157,7 +157,7 @@ export default async args => {
                     sourceMaps: !production,
                     extensions,
                     transpile: production,
-                    babelConfig: { presets: [["@babel/preset-env", { targets: { safari: "12" } }]] },
+                    babelConfig: { presets: [["@babel/preset-env", { targets: { safari: "16.3" } }]] },
                     external: commonExternalLibs
                 })
             ],
