diff --git a/configs/rollup.config.mjs b/configs/rollup.config.mjs
index b6201dbff88ca2e2ee2f67a244d7e6bb2423c132..de93dcf662e9a52e3186ee38d0405e4dfb6ccf1f 100644
--- a/configs/rollup.config.mjs
+++ b/configs/rollup.config.mjs
@@ -158,7 +158,11 @@ export default async args => {
                     minimize: production,
                     plugins: [postcssImport(), postcssUrl({ url: "inline" })],
                     sourceMap: !production ? "inline" : false,
-                    use: ["sass"]
+                    use: {
+                        sass: {
+                            silenceDeprecations: ['legacy-js-api'],
+                        }
+                    },
                 }),
                 ...getCommonPlugins({
                     sourceMaps: !production,
@@ -354,7 +358,11 @@ export function postCssPlugin(outputFormat, production, postcssPlugins = []) {
             ...postcssPlugins
         ],
         sourceMap: !production ? "inline" : false,
-        use: ["sass"],
+        use: {
+            sass: {
+                silenceDeprecations: ['legacy-js-api'],
+            }
+        },
         to: join(outDir, `${outWidgetFile}.css`)
     });
 }
diff --git a/test-config/jest.config.js b/test-config/jest.config.js
index 3df733c122ed816ef62f701559362baef1936dbe..9dab9730eafb1fe1e3efe4db84ff6a8833c3811f 100644
--- a/test-config/jest.config.js
+++ b/test-config/jest.config.js
@@ -26,7 +26,7 @@ module.exports = {
         "react-hot-loader/root": join(__dirname, "__mocks__/hot")
     },
     moduleDirectories: ["node_modules", join(projectDir, "node_modules")],
-    collectCoverage: !process.env.CI,
+    // collectCoverage: !process.env.CI,
     coverageDirectory: join(projectDir, "dist/coverage"),
     testEnvironment: "jsdom"
 };
