diff --git a/dist/index.cjs b/dist/index.cjs
index 981dd27..41c4ffd 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -369,7 +369,7 @@ function isMIMETypeWildcard(v) {
 * Check if v is a file extension.
 */
 function isExt(v) {
-	return /^.*\.[\w]+$/.test(v);
+	return /^\.[^/\\?*<>|:".]+$/.test(v);
 }
 //#endregion
 //#region src/index.tsx
diff --git a/dist/index.js b/dist/index.js
index 6f653d4..0cd7295 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -342,7 +342,7 @@ function isMIMETypeWildcard(v) {
 * Check if v is a file extension.
 */
 function isExt(v) {
-	return /^.*\.[\w]+$/.test(v);
+	return /^\.[^/\\?*<>|:".]+$/.test(v);
 }
 //#endregion
 //#region src/index.tsx
diff --git a/src/utils/index.ts b/src/utils/index.ts
index 9a47004..a077b9b 100644
--- a/src/utils/index.ts
+++ b/src/utils/index.ts
@@ -610,5 +610,5 @@ export function isMIMETypeWildcard(v: string): boolean {
  * Check if v is a file extension.
  */
 export function isExt(v: string): boolean {
-  return /^.*\.[\w]+$/.test(v);
+  return /^\.[^/\\?*<>|:".]+$/.test(v);
 }
